code/game/machinery/bioprinter.dm

```diff
@@ -69,7 +70,8 @@
 		stored_matter -= products[choice][2]
 		var/new_organ = products[choice][1]
 		var/obj/item/organ/O = new new_organ(get_turf(src))
-		O.is_printed = TRUE // Heist stuff.
+		if(choice == !"tooth")
```

Comment:
If you have actual suggestions, feel free to list them. This line is merely to make it not produce runtimes. The code works fine.